regression-test/suites/nereids_tpcds_shape_sf100_p0/noStatsRfPrune/query1.groovy

```diff
@@ -45,7 +45,7 @@ where sr_returned_date_sk = d_date_sk
 and d_year =2000
 group by sr_customer_sk
 ,sr_store_sk)
- select  c_customer_id
+ select /*+ FORCE_CBO_RULE(no_push_down_agg_through_join no_push_down_agg_through_join_one_side) */  c_customer_id

```

Comment:
we should check perfermance is ok after let these rules become cost based rewrite tule

Reply:
I have changed this rule to original behavior